libraries/chain/transaction_context.cpp

```diff
@@ -321,6 +333,14 @@ namespace eosio { namespace chain {
    void transaction_context::finalize() {
       EOS_ASSERT( is_initialized, transaction_exception, "must first initialize" );
 
+      net_usage = ((net_usage + 7)/8)*8; // Round up to nearest multiple of word size (8 bytes)
+      auto now = fc::time_point::now();
+      trace->elapsed = now - start;
```

Comment:
We want `elapsed` to include as much of the transaction time as possible. There are some db lookups and modifications after this line that should be included in the `elapsed` time.